jdaviz/cli.py

```diff
@@ -65,7 +67,7 @@ def main(filename, layout='default', browser='default', theme='light',
         notebook_template = notebook_template.replace("# PREFIX", "from jdaviz import enable_hot_reloading; enable_hot_reloading()")  # noqa: E501
 
     with open(os.path.join(nbdir, 'notebook.ipynb'), 'w') as nbf:
-        nbf.write(notebook_template.replace('DATA_FILENAME', filepath).replace('JDAVIZ_VERBOSITY', verbosity).replace('JDAVIZ_HISTORY_VERBOSITY', history_verbosity).strip())  # noqa: E501
+        nbf.write(notebook_template.replace('DATA_FILENAME', filepath).replace('JDAVIZ_VERBOSITY', verbosity).replace('JDAVIZ_HISTORY_VERBOSITY', history_verbosity).replace('INSTRUMENT', instrument).strip())  # noqa: E501
```

Comment:
Will this break other configs?

Reply:
Nope! I tested `str.replace` and found it just returns the original string if it doesn't detect the search criteria